targets/PythonSdk/make.js

```diff
@@ -0,0 +1,407 @@
+    var isError = isDeprecated && (new Date() > deprecationTime) ? "true" : "false";
+
+    if (isDeprecated && apiObj.deprecation.ReplacedBy != null)
+        return tabbing + "# [Obsolete(\"Use '" + apiObj.deprecation.ReplacedBy + "' instead\", " + isError + ")]\n";
+    else if (isDeprecated)
+        return tabbing + "# [Obsolete(\"No longer available\", " + isError + ")]\n";
+    return "";
+}
+
+function generateApiSummary(tabbing, apiElement, summaryParam, extraLines) {
+    var lines = generateApiSummaryLines(apiElement, summaryParam, extraLines);
+
+    var output;
+    if (lines.length === 1) {
+        output = tabbing + "# " + lines.join("\n" + tabbing + "# ") + "\n";
+    } else if (lines.length > 0) {
+        output = tabbing + "# " + lines.join("\n" + tabbing + "# ") + "\n";
+    } else {
+        output = "";
+    }
+    return output;
+}
+
+function getComparator(tabbing, dataTypeName, dataTypeSortKey)
+{
+    //var output = multiTab(tabbing, 3) + "def __eq__(self, " + dataTypeName + " other):\n" +
+    var output = multiTab(tabbing, 2) + "def __eq__(self, other):\n" +
+        multiTab(tabbing, 3) + "if other == None or other." + dataTypeSortKey + " == None:\n" +
+        multiTab(tabbing, 4) + "return 1\n" +
+        multiTab(tabbing, 3) + "if " + dataTypeSortKey + " == None:\n" +
+        multiTab(tabbing, 4) + "return -1\n"+
+        multiTab(tabbing, 3) + "return "+dataTypeSortKey+".__eq__(self."+dataTypeSortKey+", other."+dataTypeSortKey+")\n";
+
+    return output;
+}
+
+function multiTab(tabbing, numTabs)
+{
+    var finalTabbing = "";
+    while (numTabs != 0)
+    {
+        finalTabbing += tabbing;
+        numTabs--;
+    }
+    return finalTabbing;
+}
+
+function getDefaultValueForType(property, datatype) {
+
+    if (property.jsontype === "Number")
+        return "0";
+    if (datatype.jsontype === "String")
+        return "\"\"";
+    if(datatype.JsonType === "Object" && datatype.isOptional)
+        return "None";
+    //if(datatype.JsonType === "Object")
+    //    return "new " + ?namespace ? + datatype.actualtype + "()";
+    if(datatype.JsonType === "Object")
+        return "new " + datatype.actualtype + "()";
+
+    if (property.actualtype === "String")
+        return "\"\"";
+    else if (property.actualtype === "Boolean")
+        return "False";
+    else if (property.actualtype === "int16")
+        return "0";
+    else if (property.actualtype === "uint16")
+        return "0";
+    else if (property.actualtype === "int32")
+        return "0";
+    else if (property.actualtype === "uint32")
+        return "0";
+    else if (property.actualtype === "int64")
+        return "0";
+    else if (property.actualtype === "uint64")
+        return "0";
+    else if (property.actualtype === "float")
+        return "0.0";
+    else if (property.actualtype === "double")
+        return "0.0";
+    else if (property.actualtype === "DateTime")
+        return "datetime.date.today()";
```

Comment:
I don't think today is a good datetime default.
It should just be DateTime.min or something (whatever the python equivalent is)